public/manifest.json

```diff
@@ -25,7 +25,7 @@
     "service_worker": "js/background.js"
   },
 
-  "permissions": ["storage"],
+  "permissions": ["storage", "scripting", "activeTab"],
```

Comment:
Scripting and active tab permissions needed to dynamically add content-script execution for self hosted GitHub instances.